### PR TITLE
Mejora en reporte de certificado de logro

### PR DESCRIPTION
**Pull request by aragonc**
@NicoDucou revisalo

### CODE REVIEW

**Comment by NicoDucou** [comment]
Funciona bien con el estilo Chamilo pero si se selecciona otro estilo CSS entonces sale mal.
Tu cambios no deberian ser en el estilo Chamilo especifico pero la base para que funcione con todos los CSS.

**Comment by jmontoyaa** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 12
- Added 11
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5266914&bid=17254522)

File: app/Resources/public/css/print.css [hunk 1]
```diff
@@ -119,3 +93,102 @@ table.table td {
...
+/* print table pdf */
+.data_table{
+    width: 100%;
+    padding: 0;
+    border: 1px solid #b5cae7;
+    border-spacing: 0;
+    margin: 0 0 18px;
+}
+
+.data_table td {
```
Comment (jmontoyaa) [inline app/Resources/public/css/print.css]: Codacy Issue found: [Expected selector ".data_table td" to come before selector "table.table td" (no-descending-specificity)](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5266914)

File: app/Resources/public/css/print.css [hunk 2]
```diff
@@ -119,3 +93,102 @@ table.table td {
...
+    padding: 5px;
+    border-bottom: 1px solid #b5cae7;
+    border-right: 1px solid #b5cae7;
+    border-top: 1px solid #b5cae7;
+    text-align: left;
+    font-size: 12px;
+    color: #666666;
+}
+
+.data_table th, .data_table td {
```
Comment (jmontoyaa) [inline app/Resources/public/css/print.css]: Codacy Issue found: [Expected selector ".data_table td" to come before selector "table.table td" (no-descending-specificity)](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5266914)

File: app/Resources/public/css/print.css [hunk 3]
```diff
@@ -119,3 +93,102 @@ table.table td {
     border: 1px solid #555;
     border-collapse: collapse;
 }
+
+/* print table pdf */
+.data_table{
+    width: 100%;
+    padding: 0;
+    border: 1px solid #b5cae7;
```
Comment (jmontoyaa) [inline app/Resources/public/css/print.css]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5266914)

File: app/Resources/public/css/print.css [hunk 4]
```diff
@@ -119,3 +93,102 @@ table.table td {
...
+}
+
+.table_print td {
+    padding: 5px;
+    border-bottom: 1px solid #CEE0CC;
+    border-right: 1px solid #CEE0CC;
+    border-top: 1px solid #CEE0CC;
+    text-align: left;
+    font-size: 12px;
+    color: #666666;
```
Comment (jmontoyaa) [inline app/Resources/public/css/print.css]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5266914)

File: app/Resources/public/css/print.css [hunk 5]
```diff
@@ -119,3 +93,102 @@ table.table td {
...
+    font-weight: bold;
+    background-color: #eef5fa;
+    border-bottom: 2px solid #b5cae7;
+    color: #4171b5;
+}
+.data_table th a {
+    color: #333;
+}
+.data_table_no_border {
+    border-collapse: collapse;
```
Comment (jmontoyaa) [inline app/Resources/public/css/print.css]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5266914)

File: app/Resources/public/css/print.css [hunk 6]
```diff
@@ -119,3 +93,102 @@ table.table td {
...
+    padding: 0;
+    border: 1px solid #b5cae7;
+    border-spacing: 0;
+    margin: 0 0 18px;
+}
+
+.data_table td {
+    padding: 5px;
+    border-bottom: 1px solid #b5cae7;
+    border-right: 1px solid #b5cae7;
```
Comment (jmontoyaa) [inline app/Resources/public/css/print.css]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5266914)

File: main/mySpace/access_details_session.php [hunk 7]
```diff
@@ -267,12 +267,13 @@ function getReport($userId, $from, $to, $addTime = false)
...
     $courseSessionTable = '';
     $courseSessionTableData = [];
+    $iconCourse = Display::return_icon('course.png',null,[],ICON_SIZE_SMALL);
     foreach ($report as $sessionId => $data) {
         foreach ($data['courses'] as $courseId => $courseData) {
             if (empty($courseData)) {
                 continue;
             }
-            $courseSessionTable .= Display::page_subheader3($data['name'][$courseId]);
+            $courseSessionTable .= '<div class="data-title">'.Display::page_subheader3($iconCourse.$data['name'][$courseId]).'</div>';
```
Comment (jmontoyaa) [inline main/mySpace/access_details_session.php]: Codacy Issue found: [Line exceeds maximum limit of 120 characters; contains 134 characters](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5266914)

File: main/mySpace/access_details_session.php [hunk 8]
```diff
@@ -485,12 +486,14 @@ function getReport($userId, $from, $to, $addTime = false)
         $courseSessionTable = $result['second'];
         $totalCourseSessionTable = $result['third'];
         $total = $result['total'];
-        $tableList .= '<div style="text-align:center">'.Display::page_subheader2($dateToCheck).'</div>'.$table->toHtml();
+        $iconCalendar = Display::return_icon('calendar.png',null,[],ICON_SIZE_SMALL);
+        $tableList .= '<div class="date-calendar">'.Display::page_subheader2($iconCalendar.get_lang('Date').': '.$dateToCheck).'</div>';
```
Comment (jmontoyaa) [inline main/mySpace/access_details_session.php]: Codacy Issue found: [Line exceeds maximum limit of 120 characters; contains 136 characters](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5266914)

File: app/Resources/public/css/print.css (hunk 4 above)
Comment (jmontoyaa) [inline app/Resources/public/css/print.css]: Codacy Issue found: [Expected "#666666" to be "#666" (color-hex-length)](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5266914)

File: app/Resources/public/css/print.css [hunk 9]
```diff
@@ -4,39 +4,13 @@ body {
     color: #000;
     margin: 0;
     padding: 0;
-    background-color: #fff;
+    font-family: "Arial", serif;
```
Comment (jmontoyaa) [inline app/Resources/public/css/print.css]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5266914)

File: app/Resources/public/css/print.css [hunk 10]
```diff
@@ -119,3 +93,102 @@ table.table td {
     border: 1px solid #555;
     border-collapse: collapse;
 }
+
+/* print table pdf */
+.data_table{
```
Comment (jmontoyaa) [inline app/Resources/public/css/print.css]: Codacy Issue found: [Expected single space before "{" (block-opening-brace-space-before)](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5266914)

**Comment by jmontoyaa** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 12
- Added 11
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5266914&bid=17254522)

File: app/Resources/public/css/print.css [hunk 11]
```diff
@@ -119,3 +93,103 @@ table.table td {
...
+    width: 100%;
+    padding: 0;
+    border: 1px solid #b5cae7;
+    border-spacing: 0;
+    margin: 0 0 18px;
+}
+
+.data_table td {
```
Comment (jmontoyaa) [inline app/Resources/public/css/print.css]: Codacy Issue found: [Expected selector ".data_table td" to come before selector "table.table td" (no-descending-specificity)](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5266914)

File: app/Resources/public/css/print.css [hunk 12]
```diff
@@ -119,3 +93,103 @@ table.table td {
...
+    font-weight: bold;
+    background-color: #eef5fa;
+    border-bottom: 2px solid #b5cae7;
+    color: #4171b5;
+}
+.data_table th a {
+    color: #333;
+}
+.data_table_no_border {
+    border-collapse: collapse;
```
Comment (jmontoyaa) [inline app/Resources/public/css/print.css]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5266914)

File: main/mySpace/access_details_session.php (hunk 8 above)
Comment (jmontoyaa) [inline main/mySpace/access_details_session.php]: Codacy Issue found: [Line exceeds maximum limit of 120 characters; contains 136 characters](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5266914)

File: app/Resources/public/css/print.css [hunk 13]
```diff
@@ -119,3 +93,103 @@ table.table td {
     border: 1px solid #555;
     border-collapse: collapse;
 }
+
+/* print table pdf */
+.data_table{
```
Comment (jmontoyaa) [inline app/Resources/public/css/print.css]: Codacy Issue found: [Expected single space before "{" (block-opening-brace-space-before)](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5266914)

File: app/Resources/public/css/print.css [hunk 14]
```diff
@@ -119,3 +93,103 @@ table.table td {
...
+}
+
+.table_print td {
+    padding: 5px;
+    border-bottom: 1px solid #CEE0CC;
+    border-right: 1px solid #CEE0CC;
+    border-top: 1px solid #CEE0CC;
+    text-align: left;
+    font-size: 12px;
+    color: #666666;
```
Comment (jmontoyaa) [inline app/Resources/public/css/print.css]: Codacy Issue found: [Expected "#666666" to be "#666" (color-hex-length)](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5266914)

File: app/Resources/public/css/print.css (hunk 9 above)
Comment (jmontoyaa) [inline app/Resources/public/css/print.css]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5266914)

File: app/Resources/public/css/print.css (hunk 14 above)
Comment (jmontoyaa) [inline app/Resources/public/css/print.css]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5266914)

File: app/Resources/public/css/print.css [hunk 15]
```diff
@@ -119,3 +93,103 @@ table.table td {
...
+    padding: 5px;
+    border-bottom: 1px solid #b5cae7;
+    border-right: 1px solid #b5cae7;
+    border-top: 1px solid #b5cae7;
+    text-align: left;
+    font-size: 12px;
+    color: #666666;
+}
+
+.data_table th, .data_table td {
```
Comment (jmontoyaa) [inline app/Resources/public/css/print.css]: Codacy Issue found: [Expected selector ".data_table td" to come before selector "table.table td" (no-descending-specificity)](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5266914)

File: app/Resources/public/css/print.css [hunk 16]
```diff
@@ -119,3 +93,103 @@ table.table td {
...
+    padding: 0;
+    border: 1px solid #b5cae7;
+    border-spacing: 0;
+    margin: 0 0 18px;
+}
+
+.data_table td {
+    padding: 5px;
+    border-bottom: 1px solid #b5cae7;
+    border-right: 1px solid #b5cae7;
```
Comment (jmontoyaa) [inline app/Resources/public/css/print.css]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5266914)

File: app/Resources/public/css/print.css [hunk 17]
```diff
@@ -119,3 +93,103 @@ table.table td {
     border: 1px solid #555;
     border-collapse: collapse;
 }
+
+/* print table pdf */
+.data_table{
+    width: 100%;
+    padding: 0;
+    border: 1px solid #b5cae7;
```
Comment (jmontoyaa) [inline app/Resources/public/css/print.css]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5266914)

**Pull request merged**